### PR TITLE
Updating Link

### PR DESCRIPTION
**Pull request by TokyoScarab**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/3299

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.
--

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
     - SQL Server 2014 SP1 + CU5 ([link](https://support.microsoft.com/help/3130926)), or higher / SQL Server 2012 SP2 + CU16 or higher/ SQL Server 2014 RTM + CU12 ([link](https://support.microsoft.com/help/3130923/cumulative-update-12-for-sql-server-2014)) or higher / SQL Server 2014 SP2
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
    - [SQL Server 2014 SP1 + CU5](https://support.microsoft.com/help/3130926), or higher / SQL Server 2012 SP2 + CU16 or higher / [SQL Server 2014 RTM + CU12](https://support.microsoft.com/help/3130923/cumulative-update-12-for-sql-server-2014), or higher / SQL Server 2014 SP2
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
...
      - SQL Server Native Client for SQL Server 2012 ([link](https://www.microsoft.com/download/details.aspx?id=50402))
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
     - [SQL Server Native Client for SQL Server 2012](https://www.microsoft.com/download/details.aspx?id=50402)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 3]
```diff
@@ -138,7 +138,7 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
...
      - Microsoft ODBC Driver 11 for SQL Server ([link](https://www.microsoft.com/download/details.aspx?id=36434)), or higher
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
     - [Microsoft ODBC Driver 11 for SQL Server](https://www.microsoft.com/download/details.aspx?id=36434), or higher
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 4]
```diff
@@ -138,7 +138,7 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
...
-     - Shared Management Objects for SQL Server 2014 SP2 ([link](https://www.microsoft.com/download/details.aspx?id=42295))
+     - Shared Management Objects for SQL Server 2014 SP2 ([link](https://www.microsoft.com/download/details.aspx?id=53164))
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
     - [Shared Management Objects for SQL Server 2014 SP2](https://www.microsoft.com/download/details.aspx?id=53164)
```

File: Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md [hunk 5]
```diff
@@ -138,7 +138,7 @@ Extensive dependency updating is required before you begin to disable TLS 1.0 an
...
      - SQLSysClrTypes for SQL server 2014 SP2 ([link](https://www.microsoft.com/download/details.aspx?id=42295))
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/manage/topology/disable-tls-1.0-1.1.md]:
```suggestion
     - [SQLSysClrTypes for SQL server 2014 SP2](https://www.microsoft.com/download/details.aspx?id=42295)
```

**Comment by scanum** [comment]
@LanaChin 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**